test/pod.bats

```diff
@@ -287,3 +287,43 @@ function teardown() {
 	cleanup_pods
 	stop_crio
 }
+
+@test "invalid systemd cgroup_parent fail" {
+	if [[ "$CGROUP_MANAGER" != "systemd" ]]; then
+		skip "need systemd cgroup manager"
+	fi
+
+	wrong_cgroup_parent_config=$(cat "$TESTDATA"/sandbox_config.json | python -c 'import json,sys;obj=json.load(sys.stdin);obj["linux"]["cgroup_parent"] = "podsandbox1.slice:container:infra"; json.dump(obj, sys.stdout)')
+	echo "$wrong_cgroup_parent_config" > "$TESTDIR"/sandbox_wrong_cgroup_parent.json
+
+	start_crio
+	run crioctl pod run --config "$TESTDIR"/sandbox_wrong_cgroup_parent.json
+	echo "$output"
+	[ "$status" -eq 1 ]
+	[[ "$output" =~ "expected cgroupsPath to be of format \"slice:prefix:name\" for systemd cgroups" ]]
```

Comment:
@runcom @mrunalp yes, this is a libcontainer specific string so we probably want to only check the exit status of the command.